src/leappto/cli.py

```diff
@@ -51,7 +51,7 @@
     _set_inheritable = None
 
 # Checking for required permissions
-_REQUIRED_GROUPS = ["vagrant", "libvirt"]
+_REQUIRED_GROUPS = ["vagrant"]
```

Comment:
@ncoghlan Yes I want to get rid of that too, I am just trying to verify it before ;-)